src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -22,7 +20,7 @@
 @CommandLineProgramProperties(summary="Align assembled contigs to the reference and call breakpoints from them.",
         oneLineSummary="Align contigs and call breakpoints",
         programGroup = SparkProgramGroup.class)
-public class AlignContigsAndCallBreakpointsSpark extends GATKSparkTool {
+public class AlignAssembledContigsSpark extends GATKSparkTool {
```

Comment:
If you think so, the other two (`CallVariantsFromAlignedContigsSpark` and `CallVariantsFromAlignedContigsSAMSpark`) should be marked accordingly as well.